docker/from-packages/scripts/start-collabora-online.pl

```diff
@@ -34,9 +35,42 @@ ()
 
                 $output .= "                </group>\n";
             }
+        } elsif (/^domain/) {
+            my $value = $ENV{$_};
+            $message .= "WARNING: The 'domain=$value' is deprecated.\n For your convenience, we interpret it as the following:\n";
+            my @hosts = split('\|', $value);
+            if (@hosts) {
```

Comment:
I suppose the WARNING should read a bit differently when the domain= is completely empty (ie. if (@hosts) is false)?